<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Drawer" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>781</width>
        <height>384</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="action_reset_all_data"/>
    <addaction name="separator"/>
    <addaction name="action_insert_man"/>
    <addaction name="action_insert_woman"/>
    <addaction name="separator"/>
    <addaction name="action_insert_men"/>
    <addaction name="action_insert_women"/>
    <addaction name="separator"/>
    <addaction name="action_set_priorities_of"/>
    <addaction name="action_generate_priorities"/>
    <addaction name="separator"/>
    <addaction name="action_solve_the_problem"/>
    <addaction name="action_solve_the_problem_step_by_step"/>
   </widget>
   <addaction name="menuOperations"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_insert">
   <property name="text">
    <string>Insert nodes</string>
   </property>
  </action>
  <action name="action_draw">
   <property name="text">
    <string>Draw ledges</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="text">
    <string>Calculate</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="action_insert_man">
   <property name="text">
    <string>Insert men one by one</string>
   </property>
  </action>
  <action name="action_insert_woman">
   <property name="text">
    <string>Insert woman one by one</string>
   </property>
  </action>
  <action name="action_reset_all_data">
   <property name="text">
    <string>Reset all data</string>
   </property>
  </action>
  <action name="action_set_priorities_of">
   <property name="text">
    <string>Set priorities of single element</string>
   </property>
  </action>
  <action name="action_set_max_prio_length">
   <property name="text">
    <string>Set max prio length</string>
   </property>
  </action>
  <action name="action_solve_the_problem_step_by_step">
   <property name="text">
    <string>Solve the problem step by step</string>
   </property>
  </action>
  <action name="action_solve_the_problem">
   <property name="text">
    <string>Solve the problem</string>
   </property>
  </action>
  <action name="action_insert_men">
   <property name="text">
    <string>Insert men automatically</string>
   </property>
  </action>
  <action name="action_insert_women">
   <property name="text">
    <string>Insert women automatically</string>
   </property>
  </action>
  <action name="action_generate_priorities">
   <property name="text">
    <string>Generate priorities of elements</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Drawer</class>
   <extends>QWidget</extends>
   <header location="global">drawer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
